apps/vr-tests/src/stories/Stack.stories.tsx

```diff
@@ -19,11 +23,11 @@ const itemStyles = {
 
 const boxStyles: IStyle = {
   ...itemStyles,
-  display: 'flex',
-  justifyContent: 'center' as 'center',
-  alignItems: 'center' as 'center',
-  width: '50px',
-  height: '50px',
+  display: "flex",
```

Comment:
You have some strange prettier rules here I think?

Reply:
Yeah this doesn't seem right. Did your prettier precommit hook break?

Reply:
Yeah, I think the prettier precommit hook broke, will try running prettier again over this file.

Reply:
Should be fixed.